client/js/screens/Heroes/HeroesContainer.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import { View } from "react-native";
+import Heroes from "./Heroes";
+
+export default class HeroesContainer extends Component {
+  static navigationOptions = {
+    title: "Heroes"
+  };
+
+  render() {
+    console.log(this.props);
```

Comment:
remove console log.

Reply:
**Suggestion**: Instead of using `console.log`, add a file in `util/index.js`
Inside, add a debug function like 
```
function debug(...args) { 
  console.log(...args);
}
```

This gives you the flexibility to turn off `console.log` in one place.